src/08_comprehensions.py

```diff
@@ -11,6 +11,8 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
+for n in range(5):
+    y.append(n+1)
 
```

Comment:
could also be written this way
y = [n for n in range(1, 6)]
print(y)

Reply:
You showed an understanding of git workflow by committing frequently.
You were able to answer the question accurately. My meeting the MVP.
Overall you did great.